shell/components/form/GithubPicker.vue

```diff
@@ -1,343 +1,355 @@
 <script>
-import { mapState } from 'vuex';
+import SortableTable from '@shell/components/SortableTable';
+import RadioButton from '@components/Form/Radio/RadioButton';
 import debounce from 'lodash/debounce';
-import { findBy } from '@shell/utils/array';
-import { EXTENDED_SCOPES } from '@shell/store/github';
+import { isArray } from '@shell/utils/array';
 import LabeledSelect from '@shell/components/form/LabeledSelect';
-
-export const FILE_PATTERNS = {
-  dockerfile: /^Dockerfile(\..*)?$/i,
-  riofile:    /^Riofile(\..*)?$/i,
-  yaml:       /^.*\.ya?ml?$/i,
-};
+import LabeledInput from '@components/Form/LabeledInput/LabeledInput';
+import some from 'lodash/some';
 
 export default {
-  components: { LabeledSelect },
-  props:      {
-    value: {
-      type:     Object,
-      required: true,
-    },
-
-    // Filter files displayed in dropdown by keys in FILE_PATTERNS
-    filePattern: {
-      type:    String,
-      default: null,
-    },
-
-    preferredFile: {
-      type:    String,
-      default: null,
-    },
-
-    fileKey: {
-      type:    String,
-      default: 'dockerfile',
-    }
+  components: {
+    LabeledSelect,
+    SortableTable,
+    RadioButton,
+    LabeledInput,
   },
 
   data() {
-    return {
-      repos:       [],
-      branches:    [],
-      files:       [],
+    const eventHeaders = [
+      {
+        name:  'index',
+        label: 'Chose',
+      },
+      {
```

Comment:
There's a `Link` formatter (./shell/components/formatter/Link.vue) that can be used here instead of having a custom template within the list (there's an example in ./shell/edit/networking.k8s.io.ingress/Certificates.vue). If you need to pass in a `value` that has better properties for the formatter to use you can add a `getValue` property to the header definition to convert the `row` value into a nicer object